.github/ISSUE_TEMPLATE.md

```diff
@@ -8,7 +8,11 @@ This template is for bug reports. If you are reporting a bug, please continue on
 Note that leaving sections blank will make it difficult for us to troubleshoot and we may have to close the issue.
 -->
 
-**What version of ESLint are you using?**
+**Tell us about your environment**
+
+* **ESLint Version:**
+* **Node Version:**
+* **npm Version:**
```

Comment:
Do we really need Node version and NPM version? We are sometimes very strict about requiring all of the fields in the template be filled out, and those two might make a difference, but only very occasionally. For most bugs they don't really matter.